content/departments/product-engineering/process/headcount-planning.md

```diff
@@ -1,23 +1,21 @@
 # Product/Engineering headcount planning
 
-The source of truth for current and future product and engineering headcount is [this spreadsheet](https://docs.google.com/spreadsheets/d/1CIQYQDN2KFyHMmPEx3FqubapyXyapFp0B_DoDJtWvm8/edit#gid=0).
+Our headcount planning philosophy balances three things:
 
-Access is currently restricted to product/eng directors, engineering managers, and relevant people in people ops, talent, and finance.
+- Our need to have a stable financial plan for the fiscal year
+- Our desire to be flexible so we can respond to changing business needs as the fiscal year progresses
+- The lead time we need to make a great hire for a specific role
 
-## Philosophy
+## FY23 philosophy
```

Comment:
Since this page is in the process section, I wonder if the FY23 part belongs better in https://handbook.sourcegraph.com/departments/product-engineering/strategy-goals/?